resources/server/bin/helpers/check-requirements-linux.sh

```diff
@@ -66,7 +66,7 @@ if [ -n "$(ldd --version | grep -v musl)" ]; then
         libc_path='/usr/lib/libc.so.6'
     elif [ -f /sbin/ldconfig ]; then
         # Look up path
-        libc_paths=$(ldconfig -p | grep 'libc.so.6')
+        libc_paths=$(/sbin/ldconfig -p | grep 'libc.so.6')
```

Comment:
Can you also make the change at L35 https://github.com/microsoft/vscode/blob/969393c1a6d24c25e72cb7ba4f918cb13b9263a2/resources/server/bin/helpers/check-requirements-linux.sh#L35